tests/examples/test_examples.py

```diff
@@ -0,0 +1,92 @@
+import glob
+import os
+import subprocess
+import sys
+
+import pytest
+
+NO_EXAMPLES = "no examples found"
+
+
+def can_import_unstructured():
+    try:
+        import unstructured  # noqa: F401
+
+        return True
+    except ImportError:
+        return False
+
+
+get_started_examples = [
+    filename
+    for filename in glob.glob("examples/get_started/**/*.py", recursive=True)
+    # torch-loader will not finish within an hour on Linux runner
```

Comment:
models?

Reply:
yeah, that example is training a torch model. From earlier tests, it did not finish within an hour on the Linux runners.